jsapi/externs/luxx.js

```diff
@@ -3,6 +3,59 @@
  */
 var luxx;
 
+/**
+ * Object literal with config options for the map.
+ * @typedef {{
+ *     bgLayer: (string),
+ *     layers: (Array<string>),
+ *     target: (Element|string),
+ *     position: (ol.Coordinate|undefined),
+ *     positionSrs: (string|undefined),
```

Comment:
I think it would be safier if we also accept `number` for `positionSrs` and `layers`.